docs/resources/maintenance_window.md

```
@@ -10,7 +9,103 @@ description: |-
+
+## Maintenance Window Types
+
+- `once` - One-time maintenance window
+- `daily` - Daily recurring maintenance window
+- `weekly` - Weekly recurring maintenance window
+- `monthly` - Monthly recurring maintenance window
+
+## Time Format
+
+Time should be specified in `HH:MM` format (24-hour format):
+- `09:00` - 9:00 AM
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)